sbnanaobj/StandardRecord/SREXTInfo.h

```diff
@@ -0,0 +1,18 @@
+#ifndef SREXTInfo_h
+#define SREXTInfo_h
+
+namespace caf
+{
+  class SREXTInfo
+  {
+  public:
+    SREXTInfo();
+    unsigned int gates_since_last_trigger;
+    bool isBNBOffBeam;
+    bool isNuMIOffBeam;
+    bool isMajority;
+    bool isMinBias;
```

Comment:
These should be commented what they mean or their name should be changed to something more generic.